docs/user_guide/gui_features.rst

```diff
@@ -61,19 +61,19 @@ PCell shape is updated after each change in the parameters.
 Converting elements placed in GUI into code
 -------------------------------------------
 
-KQCircuits includes a macro ``export_cells_as_code.lym`` that can be used to
+KQCircuits includes a macro `export_cells_as_code.lym <https://github.com/iqm-finland/KQCircuits/blob/main/klayout_package/python/scripts/macros/export/export_cells_as_code.lym>`_ that can be used to
 convert elements placed in GUI into code that can be used in chip PCells. For
 more information, see the docstring and comments in the macro. The following
 animation demonstrates how a chip is modified by placing elements in the GUI
 and how to use this macro to include those elements in PCell generation code:
 
 .. image:: ../images/gui_workflows/converting_gui_elements_to_code.gif
 
-The macro has special handling for waveguides (of ``WaveguideCoplanar`` type).
+The macro has special handling for waveguides (of `WaveguideCoplanar <https://github.com/iqm-finland/KQCircuits/blob/main/klayout_package/python/kqcircuits/elements/waveguide_coplanar.py>`_ type).
```

Comment:
Here it would be more useful to link to the API documentation of WaveguideCoplanar class, which can be done like this:
```suggestion
The macro has special handling for waveguides (of :class:`.WaveguideCoplanar` type).
```